include/matx/core/type_utils.h

```diff
@@ -52,6 +52,10 @@
 
 namespace matx {
 
+enum {
+  matxNoRank = -1
```

Comment:
shouldn't this be defined with the rest of the things like `matxEnd`?

Reply:
I don't think so.  matxEnd is a different concept entirely used for clone.  this has nothing to do with clone.